File: Python/PracticaBasica/Practica1.py
import random


numeros = [random.randint(-10, 10) for _ in range(10)]

cuentaNegativo = 0
cuentaPositivo = 0
cuentaCero = 0


for numero in numeros:
    if numero > 0:
        cuentaPositivo += 1
    elif numero < 0:
        cuentaNegativo += 1
    else:
        cuentaCero += 1


print("Números:", numeros)
print("El total de números positivos es:", cuentaPositivo)
print("El número total de números negativos es:", cuentaNegativo)
print("El número total de ceros es:", cuentaCero)

File: Java/PracticaBasica/Practica1.java
package Java.PracticaBasica;

import java.util.Scanner;

public class Practica1 {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        int [] numeros = new int[10];

        for(int i = 0; i <=numeros.length; i++) {
            System.out.println("Ingrese un numero: ");
            numeros[i] = sc.nextInt();
            // numeros[i] = (int)(Math.random()*100);
            // System.out.println(numeros[i]);
        }

        

        int cuentaPositivo = 0;
        int cuentaNegativo = 0;
        int cuentaCero = 0;

        for(int i = 0; i<numeros.length; i++) {
            if (numeros[i] > 0) {
                cuentaPositivo++;
            }else if(numeros[i] < 0) {
                cuentaNegativo++;
            }else{
                cuentaCero++;
            }
        }

        System.out.println("El total de numeros positivos es: "+cuentaPositivo);
        System.out.println("El numero total de numeros negativos es: "+cuentaNegativo);
        System.out.println("El numero total de ceros es: "+cuentaCero);
        
    }
    
}
